py_mind_memo/graphics.py

```
@@ -694,4 +743,20 @@ def clear(self):
         self.text_items.clear()
         self.line_items.clear()
         self.reference_items.clear()
+        self.image_items.clear()
+        self.icon_items.clear()
+        
+        for photo, _ in self.image_cache.values():
+            try:
+                photo.__del__()
+            except Exception:
+                pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)